docs/getting-started/prerequisites.md

```diff
@@ -61,6 +61,7 @@ If you already have Python installed, you can verify its version by running
 Install the python dependencies by running the following command:
 
 ```bash
+sudo apt-get install python3-dev python3-venv
```

Comment:
You needed this to get things running right?

Reply:
Yes, otherwise the next step will report things missing. In fact, I first install ``python3-venv`` and then ``python3-dev``, according to reported error messages.